specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json

```diff
@@ -2048,6 +2071,31 @@
               "type": "string",
               "description": "Impacted resource region name."
             },
+            "resourceName": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource name."
```

Comment:
nit: resource name of the impacted resource.

Reply:
are you referring to the resource id here? if yes, it can be formatted as arm-id